src/App.tsx

```diff
@@ -1,24 +1,30 @@
-// import { AboutMe } from './components/AboutMe/AboutMe.tsx';
 import { useState } from 'react';
 
 import { AboutMe } from './components/AboutMe/AboutMe.tsx';
 import { Footer } from './components/Footer/Footer.tsx';
-import { HeaderComponent } from './components/HeaderComponent/HeaderComponent.tsx';
+import { Header } from './components/Header/Header.tsx';
 import { ProductsList } from './components/ProductsList/ProductsList.tsx';
 import { PRODUCTS_DATA } from './data/data.ts';
 import type { Product } from './interfaces/Product.ts';
 
 // import styles from './App.module.css';
 
-export type Pages = 'about' | 'products';
+export enum PagesEnum {
```

Comment:
move enum into separate file, for constans, you are using this enum in a several files, so it will be wise to create a separate module for it, suggest u to do it in the same way, as u did with interfaces

do not use data type as a variable name, rename PagesEnum => Pages, or create another name for this enum